Linux build failure

Trying to build this as per the instructions in the README yields this error:
```
/tmp/couchbase-lite-C/src/CBLLog.cc: In function ‘void CBLLog_SetCallback(CBLLogCallback)’:
/tmp/couchbase-lite-C/src/CBLLog.cc:86:36: error: operands to ‘?:’ have different types ‘CBLLog_SetCallback(CBLLogCallback)::<lambda(C4LogDomain, C4LogLevel, const char*, __va_list_tag*)>’ and ‘std::nullptr_t’
   86 |                           callback ? c4Callback : nullptr,
      |                           ~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
```

GCC version: 10.1.0
CMake version: 3.17.3
ICU version: 67.1
Linux version: 5.7.4-arch x86_64

--- a/src/CBLLog.cc
+++ b/src/CBLLog.cc
@@ -83,7 +83,7 @@ void CBLLog_SetCallback(CBLLogCallback callback) CBLAPI {
 
     sCallback = callback;
     c4log_writeToCallback(c4log_callbackLevel(),
-                          callback ? c4Callback : nullptr,
+                          callback ? c4Callback :  (C4LogCallback) nullptr,
                           true);
 }
 

